internal/cmd/stack/run_prioritize.go

```diff
@@ -0,0 +1,52 @@
+package stack
+
+import (
+	"fmt"
```

Comment:
[imports not sorted?](https://github.com/spacelift-io/spacectl/blob/89a33d36e07ed88308b1fac96ab1dbdfd90886d3/internal/cmd/stack/local_preview.go#L3)

Reply:
Maybe I should add the golangci-linter config to detect it in ci?

Reply:
https://github.com/spacelift-io/spacectl/pull/244